### PR TITLE
JSUI-2367 Facet search accessibility: toggle aria attribute when elements are shown

### PR DESCRIPTION
**Pull request by ThibodeauJF**
https://coveord.atlassian.net/browse/JSUI-2367

[Deploy](https://dashboard.heroku.com/pipelines/a3535101-5bbf-4a5b-a909-47fcf8c9f149)

### CODE REVIEW

**Comment by ThibodeauJF** [comment]
@olamothe in the AccessibilityFacet tests, the `search should be accessible`
there is the focus on the facetSearch and then the testing of the results.
I can add one that blurs the facetSearch then retest to check that it's gone

File: src/ui/Facet/FacetSearchElement.ts [hunk 1]
```diff
@@ -7,25 +7,27 @@ import { EventsUtils } from '../../utils/EventsUtils';
 import { PopupUtils, PopupHorizontalAlignment, PopupVerticalAlignment } from '../../utils/PopupUtils';
```
Comment (samisayegh) [inline src/ui/Facet/FacetSearchElement.ts]: I inspected the element while NVDA is active as you asked JF. Here is what I hear when I click enter on the facet search div:

```
Combobox expanded. Search edit has autocomplete. Blank. List.

Select task with 140 results. 140 Task exclude Task. 1 of 15.
```

When using the arrow keys to navigate, only the second line is read.

```
Select <facet value> with <count> results. <count> <facet value> exclude <facet value>. 1 of 15.
```

Is that similar to what you hear on Mac?

Comment (ThibodeauJF) [reply src/ui/Facet/FacetSearchElement.ts]: Yep that's sound right! Thanks a lot Sami!

File: src/ui/Facet/FacetSearchElement.ts [hunk 2]
```diff
@@ -276,4 +279,14 @@ export class FacetSearchElement {
       this.facetSearchUserInputHandler.handleFacetSearchResultsScroll();
     }
   }
+
+  private toggleAriaAttributes(enable: boolean) {
+    if (!this.input || !this.combobox) {
+      return;
+    }
+
+    $$(this.combobox).setAttribute('role', enable ? 'combobox' : '');
```
[the remark this answers is not in the document]
Comment (ThibodeauJF) [reply src/ui/Facet/FacetSearchElement.ts]: Accessibility Tests should be sufficient.

**Pull request merged**